lib/src/widgets/utils/ds_user_avatar.widget.dart

```diff
@@ -3,24 +3,30 @@ import 'package:flutter/material.dart';
 
 import '../../themes/colors/ds_colors.theme.dart';
 import '../../themes/icons/ds_icons.dart';
+import '../../themes/texts/styles/ds_body_text_style.theme.dart';
 import '../../utils/ds_utils.util.dart';
-import '../texts/ds_body_text.widget.dart';
+import '../texts/ds_text.widget.dart';
 
 class DSUserAvatar extends StatelessWidget {
   final String? text;
   final Uri? uri;
   final double radius;
   final Color backgroundColor;
   final Color textColor;
+  final TextStyle textStyle;
 
-  const DSUserAvatar({
+  DSUserAvatar({
     super.key,
     this.text,
     this.uri,
     this.radius = 25.0,
     this.backgroundColor = DSColors.primaryGreensTrue,
-    this.textColor = Colors.black,
-  });
+    this.textColor = DSColors.neutralDarkCity,
+    TextStyle textStyle = const DSBodyTextStyle(),
+  }) : textStyle = textStyle.apply(
```

Comment:
Since ```TextStyle``` already has a ```color``` parameter, you could get rid of the ```textColor``` prop to avoid duplicity. Then, my suggestion would be to use the [Merge](https://api.flutter.dev/flutter/painting/TextStyle/merge.html) method from ```TextStyle``` so you can assure the usage of default values in case they aren't passed through ```textStyle``` prop:

```dart
static const _defaultTextStyle = TextStyle(
    color: DSColors.neutralDarkCity,
    overflow: TextOverflow.clip,
  );
```

```dart
: textStyle = _defaultTextStyle.merge(textStyle)
```

Reply:
Done!